parlai/core/testing_utils.py

```diff
@@ -179,18 +179,36 @@ def capture_output():
     Use as a context manager.
 
     >>> with capture_output() as output:
-    ...     logger.info('hello')
+    ...     print('hello')
     >>> output.getvalue()
     'hello'
     """
     sio = TeeStringIO()
-    # logger.mute()  # Stop logging to stdout
-    # logger.redirect_out(sio)  # Instead log to sio (to preserve output for later)
     with contextlib.redirect_stdout(sio), contextlib.redirect_stderr(sio):
         yield sio
-    # yield sio  # Uncomment this once all print statements removed and comment the above two statements instead
-    # logger.stop_redirect_out()  # Stop redirecting [Removes handler]
-    # logger.unmute()  # From now on log to stdout
+
+
+# # TODO: Replace capture_output with this version before completing #2044
```

Comment:
In the future i'd prefer if you didn't land commented out code. Please make an appropriate branch.

Reply:
Discussed this aspect with @klshuster and this is the solution we came up with. But can keep that in mind for the future.